dbms/src/test/java/org/polypheny/db/cypher/DmlInsertTest.java

```diff
@@ -104,9 +104,9 @@ assert containsNodes( res, true,
 
 
     @Test
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520461669